src/panels/config/logs/show-dialog-download-logs.ts

```diff
@@ -0,0 +1,18 @@
+import { fireEvent } from "../../../common/dom/fire_event";
+
+export interface DownloadLogsDialogParams {
+  header?: string;
+  provider: string;
+  numberOfLines?: number;
```

Comment:
```suggestion
  defaultLineCount?: number;
```
As it's only the default value, right?